src/Components/Dropdowns/Composed/MenuDropdown.tsx

```diff
@@ -329,10 +339,11 @@ export const MenuDropdown: FC<MenuDropdownProps> = ({
           <List
             className="List"
             height={150}
-            itemCount={queryResults.length}
+            itemCount={queryResults.length + 1}
             itemSize={50}
             width={menuWidth}
-            itemData={queryResults}
+            /* Multi-org UI tickets #4051 and #4047: display the currently selected item first. */
+            itemData={[selectedOption, ...queryResults]}
```

Comment:
instead of overwriting the list, we can use the `scrollToItem` property of this component. 

https://react-window.vercel.app/#/api/FixedSizeList:~:text=scrollToItem(index%3A%20number%2C%20align%3A%20string%20%3D%20%22auto%22)%3A%20void

Reply:
What do you think about that?

Reply:
Hm - so, that's an option, but since it would mean the item isn't displaying at the top, we should ask @Luckette . The reason I've done it this way is so the item is always at the top - it's pulled out of the array during the filter process, then pushed back in as item `[0]` when displayed to the user.

"The currently selected account should always display at the top of the dropdown regardless of the ABC order so that the user can quickly see which account is currently selected."

Reply:
Okay okay that makes sense, thank you for explaining :)

Reply:
@ChitlangeSahas I thought about this a little more, and figured out a better way to do it in the same `sort` the UI is already doing - so removed this change from clockface.

Reply:
Thanks for updating me of the change !